android/build.gradle

```
@@ -5,7 +5,7 @@ buildscript {
         buildToolsVersion = "29.0.2"
         minSdkVersion = 21
         compileSdkVersion = 29
```

[RNUH](https://react-native-community.github.io/upgrade-helper/?from=0.64.1&to=0.66.2) recommends setting `compileSdkVersion` to 30.
```suggestion
        compileSdkVersion = 30
```
Normally `compileSdkVersion` is allowed to lag behind `targetSdkVersion` if you don't use new API features from the target version when developing native Java Android apps, but since it's managed by RN here it's better to match them.

---

It's set to 31. Don't know by whom?

---

That was an old comment. Karl bumped up both versions to 31 in the notifications PR, let's keep them at 31.